transcom/mymove
# B-18269 show estimated on payment request

## [B-18269](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A873140)

## Summary

Allow TIO to view PPM Weights

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Find PPM with payment request
2. log in as TIO
3. view Payment request page 
4. verify estimated and Actual(if available) Weight displays

### Frontend



## Screenshots
![image](https://github.com/transcom/mymove/assets/136509766/7808f9bd-e39b-4fba-b3be-c9f1879b55f7)
updated to:

![image](https://github.com/transcom/mymove/assets/136509766/80c24aed-65fb-4c80-a854-20f7b9078c8c)
